lib/schemas/index.js

```diff
@@ -8,6 +8,8 @@ const ajv = new Ajv({
 
 const registeredKeys = [];
 
+require('ajv-keywords')(ajv, 'instanceof');
```

Comment:
This should be happening just after `ajv` has been instantiated.

Reply:
Noted.